### PR TITLE
Implement CLI config flags, cleanup logs

### PR DESCRIPTION
**Pull request by Djadih**
Implement CLI config flags, cleanup logs

### CODE REVIEW

**Comment by Djadih** [comment]
Demo of use
```./build/bin/quai-stratum -config config/config.json -zone 8610 -stratum 3332```

File: api/server.go [hunk 1]
```diff
@@ -212,7 +212,7 @@ func (s *ApiServer) purgeStale() {
 }
 
 func (s *ApiServer) collectStats() {
-	start := time.Now()
+	// start := time.Now()
```
Comment (wizeguyy) [inline api/server.go:214]: No dead code please. Just delete the lines.

File: api/server.go [hunk 2]
```diff
@@ -58,13 +58,12 @@ type Entry struct {
...
 
-	for level := 0; level < common.HierarchyDepth; level++ {
-		rpcConfig := settings["Upstream"].([]interface{})[level].(map[string]interface{})
-		rpcDaemons[level] = rpc.NewRPCClient(
-			rpcConfig["Name"].(string),
-			rpcConfig["Url"].(string),
-			rpcConfig["Timeout"].(string))
-	}
+	rpcConfig := settings["Upstream"].([]interface{})[0].(map[string]interface{})
+	rpcDaemons[0] = rpc.NewRPCClient(
```
Comment (wizeguyy) [inline api/server.go:62]: What is magic number 0 here? Can we make it non-magical? I strongly prefer muggle numbers

File: proxy/proxy.go [hunk 3]
```diff
@@ -63,14 +63,11 @@ func NewProxy(cfg *Config, backend *storage.RedisClient) *ProxyServer {
...
 
-	for level := 0; level < common.HierarchyDepth; level++ {
-		// Eventually we should verify with the "name" that it's in the correct order.
-		proxy.upstreams[level] = rpc.NewRPCClient(
-			cfg.Upstream[level].Name,
-			cfg.Upstream[level].Url,
-			cfg.Upstream[level].Timeout,
-		)
-	}
+	proxy.upstreams[0] = rpc.NewRPCClient(
```
Comment (wizeguyy) [inline proxy/proxy.go:66]: Same question about magic numbers

File: api/server.go (hunk 2 above)
Comment (hubchub) [reply api/server.go:62]: I am going to steal 'muggle' numbers for myself

**Comment by Djadih** [comment]
Demo of use
```./build/bin/quai-stratum -config config/config.json -prime 8546 -region 8578 -zone 8610 -stratum 3333```

**Review by hubchub — APPROVED** [review]

**Pull request merged**